transports/sql/index_alwaysencrypted_sqltransport_[6.2,).partial.md

```diff
@@ -2,7 +2,7 @@
 
 The SQL Server transport has support for [SQL Server Always Encrypted](https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/always-encrypted-database-engine).
 
-Note: Since Always Encrypted support only works with `Microsoft.Data.SqlClient`, this feature is only supported by the `NServiceBus.Transports.SqlTransport` package and not by the `NServiceBus.SqlTransport` package.
+Note: Since Always Encrypted support only works with `Microsoft.Data.SqlClient`, this feature is only supported by the `NServiceBus.Transport.SqlServer` package and not by the `NServiceBus.SqlServer` package.
```

Comment:
```suggestion
Note: Since Always Encrypted support only works with `Microsoft.Data.SqlClient`, this feature is only supported by the `NServiceBus.Transport.SqlServer` package.
```